docs/userguide/networking/overlay-security-model.md

```diff
@@ -0,0 +1,86 @@
+
+If you haven't alerady, read [How nodes work](../../swarm/how-swarm-mode-works/nodes.md)
+to learn about swarm mode manager nodes and worker nodes.
+
+Two key architectural components provide the enhanced security of swarm mode:
+
+* **Unidirectional manager to worker communication model.**  Swarm mode worker
+nodes are dumb agents that execute tasks assigned to them by a manager node.
+Workers can never make requests to managers. Therefore the compromise of a
+worker host machine does not allow an attacker to read or alter the swarm
+state, which is known only to managers.
+
+* **Workers connect to overlay networks on a need-to-know basis.** Only the
+worker nodes that run services employing an overlay network can access other
+nodes on the network.
+
+    For example, imagine you have a swarm with 100 worker nodes that runs
+    multiple services. You can constrain the microservice that provides credit
+    card payments to five workers that partipate in an overlay network called
+    `payment-card-net`. For infromation about constraints, refer to the
+    `docker service create` [command line reference](../../reference/commandline/service_create.md#specify-service-constraints).
+
+    In this case, only the five worker nodes on `payment-card-net` exchange
+    overlay network gossip information about the network. The remaining 95
+    workers in the swarm have no access to this network.  If one of those 95
+    nodes becomes compromised, it cannot be used to connect to the
+    `payment-card-net` network.
+
+    Manager nodes have access to all swarm mode overlay networks and should be
+    tightly secured against compromise.
+
+## Swarm mode overlay networks and unmanaged containers
+
+One consequence of this model is that if the Engine is running in swarm mode, you can't run an unamanged conatiner on the swarm mode overlay network:
+
+```bash
+$ docker run --network my-multi-host-network nginx
+
+docker: Error response from daemon: swarm-scoped network
+(my-multi-host-network) is not compatible with `docker create` or `docker
+run`. This network can only be used by a docker service.
+```
+
+This behavior is by design to achieve the security benefits described above. To
+workaround this situation, migrate the unmanaged containers to managed services
+as below:
+
+```bash
+docker service create --network my-multi-host-network my-image
+```
+
+Because swarm mode is an optional feature, the Docker Engine preserves backward
+compatibility. You can continue to rely on a third-party key-value store to
+support overlay networking if you wish.  However, switching to swarm-mode is
+strongly encouraged. In addition to the security benefits described in this
+article, swarm mode enables you to leverage the substantially greater
+scalability provided by the new services API.
+
+# Encryption and overlay networks
+
+The overlay driver does not encrypt the vxlan traffic between the nodes by
+default. To enable encryption, pass the `--opt encrypted` flag:
```

Comment:
it might be worthwhile to mention that if your nodes are not all within one restricted network (ie, in 2 data centers that are not linked i.e., some in aws, some in azure) then you may want to enable encryption?

(I presume?)